src/deepEqual.js

```diff
@@ -20,7 +19,35 @@
  * @return {boolean}
  */
 function deepEqual(a, b) {
-  // write code here
+  if (a === null || b === null
+    || (typeof a === 'number' && typeof b === 'number')
```

Comment:
You just need to verify if a is number or is null,  because if a is and b isn't so they are not equal
(a === null || b === null) || typeof a === 'number'